vllm/v1/core/kv_cache_manager.py

```diff
@@ -186,11 +186,11 @@ def get_computed_blocks(self, request: Request) -> tuple[KVCacheBlocks, int]:
                 - The number of computed tokens.
         """
         # Prefix caching is disabled or
-        # When the request requires prompt logprobs, we skip prefix caching.
-        if not self.enable_caching or (
-            request.sampling_params is not None
-            and request.sampling_params.prompt_logprobs is not None
-        ):
+        # When the request requires prompt logprobs or
+        # When pooling model with all pooling
+        # we skip reading cache, but still can write to cache
+        # to accelerate following requests
+        if not self.enable_caching or request.skip_reading_cache:
```

Comment:
`skip_reading_cache` is pretty clear for me.

```suggestion
        # We skip finding the prefix cache hit when prefix caching is 
        # disabled or the request is marked as skipping kv cache read 
        # (which happens when the request requires prompt logprobs 
        # or calls a pooling model with all pooling).
        if not self.enable_caching or request.skip_reading_cache:
```

I've made the following into a sentence.
```
         # Prefix caching is disabled or
        # When the request requires prompt logprobs or
        # When pooling model with all pooling
```

Feel that we don't need to discuss "can write to cache to accelerate following requests" here. If prefix caching is disabled, we don't write to cache. If you want to emphasize this, can you put it into the comments for the `skip_reading_cache` attribute of `Request` class?